dask_ml/preprocessing/data.py

```diff
@@ -915,3 +915,62 @@ def inverse_transform(self, X):
             X[col] = series
 
         return X
+
+
+class PolynomialFeatures(skdata.PolynomialFeatures):
+
+    __doc__ = skdata.PolynomialFeatures.__doc__
```

Comment:
Since we inherit, this should be unnecessary (I may be wrong though).

Reply:
I just copied the approach from the additional transformers. However, 
```class C(skdata.PolynomialFeatures):
    pass
print(C.__doc__)
None```
shows that the doc string is not inherited for sklearn based transformers (that inherit from BaseEstimator). I also checked that for standard python classes the doc string is not passed to "children"